awslabs/performance-dashboard-on-aws
# Upgrade the display of topic areas and dashboards in the public page using cards and sorting

## Description

GTT-1945/1947: Upgrade the public page display:
 - Sort topic areas and dashboards alphabetically.
 - Use cards to display dashboards.
 - Show the last update time relative to now.
 - Upgrade how search results are displayed.

## Testing

- Go to: https://dhvdlla2yoeqf.cloudfront.net/
- Check the main public page's topic areas and dashboards.
- Search for a term (e.g. dashboard).

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
